django/db/models/query.py

```diff
@@ -1783,6 +1787,18 @@ def using(self, alias):
         clone._db = alias
         return clone
 
+    def comment(self, message):
+        """Add a comment to the query."""
+        clone = self._clone()
+        if COMMENT_FORBIDDEN_PATTERN.search(message):
+            raise ValueError(
+                "Cannot pass strings containing /* or */ to comment(). "
+                "Escape or strip these delimiters before calling comment()."
+            )
+        commented_message = f"/* {message} */".format(message=message)
```

Comment:
It's an f-string, the `.format` isn't doing anything
```suggestion
        commented_message = f"/* {message} */"
```